### ISSUE TITLE
NgNavigation methods don't reload the injected NgModules

### ISSUE THREAD

**Issue by Meir017** [bbaia/protractor-net#37]
After calling the methods ```Back()````, ````Forward()```` and ````Refresh()```` the injected modules aren't re-injected before resuming angular and therefore they cannot be used again.

**Comment by disophisis** [comment]
I've started using some modules for disabling animations today and I am seeing the same thing.  Any word on this?

**Comment by bbaia** [comment]
I could fix NgNavigation.Refresh().
Available in version 0.9.1

**Comment by disophisis** [comment]
Yeah looking at the js version of protractor this looks like how they handle it.  I didn't actually see if they handled nav like back/forward but I think this fix would probably suffice for me 👍
